opal/mca/dl/dlopen/Makefile.am

```diff
@@ -20,4 +22,4 @@ noinst_LTLIBRARIES = libmca_dl_dlopen.la
 
 libmca_dl_dlopen_la_SOURCES = $(sources)
 libmca_dl_dlopen_la_LDFLAGS = -module -avoid-version
-libmca_dl_dlopen_la_LIBADD = $(opal_dl_dlopen_LIBS)
+libmca_dl_dlopen_la_LIBADD = $(dlopen_LIBS)
```

Comment:
This should be `dl_dlopen_LIBS`, right?

(per `opal/mca/dl/dlopen/configure.m4`)

Reply:
yes i had to hand paste from the spack checkout and messed it up.